algorithms/arrays/limit.py

```diff
@@ -12,19 +12,25 @@
 Complexity = O(n)
 """
 
-def limit(arr, min_lim = None, max_lim = None):
+
+def limit(arr, min_lim=None, max_lim=None):
     result = []
-    if min_lim == None:
+    if min_lim is None and max_lim:
         for i in arr:
-            if i<= max_lim:
+            if i <= max_lim:
                 result.append(i)
-    elif max_lim == None:
+    elif min_lim and max_lim is None:
```

Comment:
@ggran-REN The syntax warnings maybe in these `if` conditions. What if `min_lim` is `0`, then the code fails.  
Consider the case where I give `limit(arr, 0, 0)`, I would expect `[]`, but will get same array instead.  
All this can simply be solved by:
```py
def limit(arr, min_lim=None, max_lim=None):
    if min_lim is None:
        min_lim = float("-infinity")
    if max_lim is None:
        max_lim = float("infinity")
    result = []
    for i in arr:
        if min_lim <= i <= max_lim:
            result.append(i)
    return result
```

Reply:
Thasks alot ：-）！
i overlooked 0 is always False in python~

Reply:
@ggran-REN I see that you are a first-time contributor, its great :).  
BTW, you can fix the code in your branch [`ggran-REN:patch-1`](https://github.com/ggran-REN/algorithms/tree/ggran-REN-patch-1)  to reflect changes in this PR. Thanks.